backend/utils/threads_utils.py

```diff
@@ -0,0 +1,43 @@
+from http.client import HTTPException
+import json
+from typing import Any, Optional
+from schema.message import Thread
+from schema.message import Message, MessageRequest
+
+from utils.db import DataStorage
+from utils.message_utils import get_message ,update_message
+from config.settings import settings
+
+# List  all messages in a thread
+
+async def get_messages_in_thread(org_id, room_id, message_id):
+
+    # fetch message parent of the thread
+    DB = DataStorage(org_id)
+    messages = await get_message(org_id, room_id, message_id)
+    threads = messages["threads"]
+
+    return threads
+
+
+async def add_message_to_thread(org_id, room_id, message_id, request):
```

Comment:
Kindly add a docstring, please.

Reply:
@Richd0tcom That would be great if you could follow the same docstring format as in threads.py

Reply:
Unfortunately I'm not with my PC right now. I made the commit with my phone. I'll make the changes first thing at day break.

Reply:
@Richd0tcom for you to make a commit using your phone without testing your code, shows that this PR is not ready and should actually be closed.

Reply:
Closed? I made the chore ones from my phone. There's nothing to test about increasing variable names and adding docstrings.
Anyway I'm changing the docstrings now.

Reply:
@zxenonx @mikengr @AI-fae Please check again thank you.